lib/dom.js

```diff
@@ -303,6 +310,10 @@ declare class FocusEvent extends UIEvent {
   relatedTarget: ?EventTarget;
 }
 
+declare class BeforeUnloadEvent extends Event {
+  returnValue: mixed;
```

Comment:
According to the linked spec, it looks like this actually has type `string`. `mixed` is perfectly safe, but we might as well go with the more precise type here. Could you make that change, then I'll make sure it gets prompt attention?